migration/v9/model/archetype.go

```diff
@@ -7,8 +7,8 @@ type Archetype struct {
 	Comments          string
 	Review            *Review            `gorm:"constraint:OnDelete:CASCADE"`
 	Assessments       []Assessment       `gorm:"constraint:OnDelete:CASCADE"`
-	CriteriaTags      []Tag              `gorm:"many2many:ArchetypeCriteriaTags"`
-	Tags              []Tag              `gorm:"many2many:ArchetypeTags"`
+	CriteriaTags      []Tag              `gorm:"many2many:ArchetypeCriteriaTags;constraint:OnDelete:CASCADE"`
+	Tags              []Tag              `gorm:"many2many:ArchetypeTags;constraint:OnDelete:CASCADE"`
```

Comment:
I assume this fixes archetype delete.

Reply:
This already exists on main so I'm not sure why it's showing up separately. I'll rebase my PR and clean it up.